contrib/autopilot/autopilot.py

```diff
@@ -863,6 +864,8 @@ def main():
                             help="Pre-defined data set for model training.")
     arg_parser.add_argument("--model", type=str, choices=sorted(MODELS.keys()),
                             help="Type of translation model to train.")
+    arg_parser.add_argument("--model-test-settings", type=str, choices=sorted(MODEL_TEST_ARGS.keys()), default=MODEL_TEST_ARGS_TRANSFORMER,
+                            help="Model test settings. Default: %(default)s.")
```

Comment:
We shouldn't need to specify this as we can use the model type as the index into `MODEL_TEST_ARGS`.